doc/audit/builtin_maps.rst

```diff
@@ -564,4 +564,51 @@ While the contents themselves are encrypted, the table is public so as to be acc
 **Value** The mechanism by which the ledger secret was recovered.
 
 .. doxygenenum:: ccf::RecoveryType
-   :project: CCF
\ No newline at end of file
+   :project: CCF
+
+``self_healing_open.nodes``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Key** Intrinsic node ID: A string which is unique to a particular node role within a cluster. Currently its IP and port.
+
+**Value** 
+
+.. doxygenstruct:: ccf::self_healing_open::NodeInfo
+   :project: CCF
+   :members:
+
+``self_healing_open.gossip``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Key** Intrinsic node ID of the source of the gossip message.
+
+**Value**
+
+.. doxygenstruct:: ccf::self_healing_open::GossipRequest
+   :project: CCF
+   :members:
+
+``self_healing_open.chosen_node``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Value** The intrinsic node ID of the chosen node. This will either be the node this node voted for, or the node that is has received an `IAmOpen` message from.
+
+``self_healing_open.votes``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Key** Intrinsic node ID of the node which has voted for this node to be opened.
+
+``selfhealingopen.sm_state``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Value** State machine state of the self-healing open protocol.
```

Comment:
If that's a ds::StateMachine, it may be nice to use doxygen to point to the enum values, we do that in other places. Very minor.